src/paperqa/readers.py

```diff
@@ -412,15 +462,19 @@ async def read_doc(  # noqa: PLR0912
                 f"|reduction=cl100k_base{enrichment_summary}"
             ),
         )
-    elif str_path.endswith(".pdf"):
+    elif str_path.endswith((".pdf", ".docx", ".xlsx", ".pptx")):
         chunked_text = chunk_pdf(
             parsed_text, doc, chunk_chars=chunk_chars, overlap=overlap
         )
+        if str_path.endswith(".pdf"):
+            algorithm_name = "overlap-pdf"
+        else:  # .docx, .xlsx, .pptx
+            algorithm_name = "overlap-office"
         chunk_metadata = ChunkMetadata(
             size=chunk_chars,
             overlap=overlap,
             name=(
-                f"paper-qa={pqa_version}|algorithm=overlap-pdf"
+                f"paper-qa={pqa_version}|algorithm={algorithm_name}"
```

Comment:
```suggestion
                f"paper-qa={pqa_version}|algorithm=overlap-document"
```

No need to use two separate suffixes (`-pdf` vs `-office`) for chunking, since they're both calling `chunk_pdf` with identical parameters. We can just have both be called `overlap-document`.

The `ParsedMetadata` (from the reader) will be different 👌